packages/marketplace/contracts/exchange/Exchange.sol

```diff
@@ -128,9 +143,18 @@ contract Exchange is Initializable, AccessControlUpgradeable, ExchangeCore, Tran
         _setDefaultFeeReceiver(newDefaultFeeReceiver);
     }
 
-    /**
-     * @dev See {IERC165-supportsInterface}.
-     */
+    // @notice Triggers stopped state.
+    function pause() external onlyRole(BACKOFFICE_ROLE) {
+        _pause();
+    }
+
+    /// @notice Returns to normal state.
+    function unpause() external onlyRole(EXCHANGE_ADMIN_ROLE) {
```

Comment:
im not sure if the same role should be able to unpause and pause.
lets discuss it

Reply:
I think pause and unpause should be the same admin, yes